Description: Update Sage's libgap to (roughly) match version of GAP in Debian
 libgap is very tightly coupled to the specific version of GAP, so
 backport Sage 7.2's libgap into this version.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/build/pkgs/libgap/checksums.ini
+++ b/sage/build/pkgs/libgap/checksums.ini
@@ -1,4 +1,4 @@
 tarball=libgap-VERSION.tar.gz
-sha1=24cc0c739ef2bb809ac5f182d889cc84b3f99ade
-md5=2b4e3adcd6e21feeeb9475e80f4e3581
-cksum=1990698227
+sha1=22d2f099d4743678bfb2bc7da7a215be3b8ec03e
+md5=3e283da0397286b98e680804d93c6e61
+cksum=159804790
--- a/sage/build/pkgs/libgap/package-version.txt
+++ b/sage/build/pkgs/libgap/package-version.txt
@@ -1 +1 @@
-4.7.9
+4.8.3
--- a/sage/src/sage/libs/gap/gap_includes.pxd
+++ b/sage/src/sage/libs/gap/gap_includes.pxd
@@ -131,7 +131,7 @@
     cdef int libGAP_LAST_TESTING_TNUM
 
 cdef extern from "gap/read.h":
-    void* libGAP_ReadEvalCommand(libGAP_Obj context)
+    void* libGAP_ReadEvalCommand(libGAP_Obj context, libGAP_UInt *dualSemicolon)
     void* libGAP_ReadEvalFile()
     void* libGAP_ReadEvalResult
     bint libGAP_READ_ERROR()
--- a/sage/src/sage/libs/gap/util.pyx
+++ b/sage/src/sage/libs/gap/util.pyx
@@ -280,7 +280,7 @@
         libgap_start_interaction(cmd)
         try:
             sig_on()
-            status = libGAP_ReadEvalCommand(libGAP_BottomLVars)
+            status = libGAP_ReadEvalCommand(libGAP_BottomLVars, NULL)
             if status != libGAP_STATUS_END:
                 libgap_call_error_handler()
             sig_off()
@@ -464,7 +464,7 @@
         libgap_start_interaction(cmd)
         try:
             sig_on()
-            status = libGAP_ReadEvalCommand(libGAP_BottomLVars)
+            status = libGAP_ReadEvalCommand(libGAP_BottomLVars, NULL)
             if status != libGAP_STATUS_END:
                 libgap_call_error_handler()
             sig_off()
